build/deploy_local.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+if [[ ! "${TAG}" ]]; then
```

Comment:
`make deploy-local-easy` already does something similar.
We should decide on one of the 2

Reply:
`make deply-local` generates an image given a tag and complains if no tag is set. `make deploy-local-easy` generates a tag from the git branch and sets it, before calling the deploy_local script. imo those two can coexist. 

The first use case is, you want to generate an image and care about the tag and want to use a custom one, or you want two images with different tags from the same branch, with only minor changes.
The second one is, you don't care about the tag and just want an image based on whatever is the current branch-name.